tests/unit_test/action/action_test.py

```
@@ -3324,13 +3331,13 @@
             status=200,
             match=[
                 responses.matchers.json_params_matcher({
-                    "text": 'What is AI?', "site": '', "topn": 1
+                    "text": 'What is AI?', "site": '', "topn": 1, "bot": bot
                 })],
         )
 
         with mock.patch.dict(Utility.environment, {'web_search': {"trigger_task": False, "url": search_engine_url}}):
             with pytest.raises(ActionFailure, match=re.escape(f"{result}")):
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)